rafx-renderer/src/renderer_thread_pool_none.rs

```diff
@@ -0,0 +1,149 @@
+use crate::{RenderFeaturePlugin, RenderFrameJob, Renderer, RendererThreadPool};
+use rafx_framework::render_features::render_features_prelude::*;
+use std::sync::Arc;
+
+#[derive(Clone)]
```

Comment:
Comments explaining that this is a "default" pool that doesn't do multithreading, and that we expect end-users to provide their own pool, which allows them to dispatch work into whatever job system/thread pool that they have.

Reply:
This pool probably shouldn't be `pub` since it's created by the `Renderer` if the user passed `None` for the `RendererThreadPool` parameter. If it's not `pub`, do you still want the comment?

The main reason why I'm asking is because `RendererThreadPool` (the trait) _does_ have an extensive comment on it.

Reply:
I think we could do a really small comment like "This is a single-threaded default implementation that is used by default. End-users can override to integrate their own preferred method of scheduling work on threads."

Reply:
Done.